src/async_wrap.cc

```diff
@@ -198,37 +200,103 @@ class PromiseWrap : public AsyncWrap {
     MakeWeak();
   }
 
+  PromiseWrap(Environment* env, Local<Object> object, double asyncId,
```

Comment:
GitHub doesn't allow to comment at line 195 above... I think `kIsChainedPromiseField` should be also removed.

Reply:
Good point. On it. 👍